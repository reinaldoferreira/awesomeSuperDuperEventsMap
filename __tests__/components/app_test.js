import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../src/components/app';

it('renders something', () => {
  const component = renderer.create(
    <App />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
