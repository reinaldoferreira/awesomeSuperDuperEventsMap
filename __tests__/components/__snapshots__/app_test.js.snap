exports[`test renders something 1`] = `
<div>
  React simple starter
</div>
`;
